requirements.txt

```diff
@@ -1 +1,2 @@
 pyyaml
+ninja
```

Comment:
That's an optional dependency, let's not add it there

Reply:
lets make it not optional, it is available on `pip` so it's trivial to install everywhere.

Reply:
Yeah, but that's like `ccache`. I don't think that's the primary way we want people to build PyTorch

Reply:
it's not like `ccache`, it's like `cmake`. It doesn't have a global cache or anything.
I think it's much better experience and dependency is easy to install.

Reply:
I know how `ninja` works, I'm only saying that it will mostly benefit developers and not people who want to build PyTorch once, and it might be simpler to route everyone through the setuptools path by default since we know this works fairly well on all platforms (including Windows).

Reply:
ok, didn't consider windows...let's remove this then